content/en/docs/Reference/operator.md

```diff
@@ -0,0 +1,50 @@
+
+## How are Operators and Controllers related?
+
+See that following the [Operator's pattern][operator-pattern]; you will create [Controllers](https://kubernetes.io/docs/concepts/architecture/controller/) which provides a reconcile function responsible for synchronizing the resources until a desired state on the cluster. For a more practical explanation, see also [What’s in a controller?](https://book.kubebuilder.io/cronjob-tutorial/controller-overview.html#whats-in-a-controller).
+
+## What is Operator Lifecycle Manager?
+
+Operator Lifecycle Manager (OLM) helps users install, update, and manage the lifecycle of all Operators and their associated services running across their clusters. This project is a component of the [Operator Framework][operator-framework], an open-source toolkit to manage Kubernetes native applications, called Operators, in an effective, automated, and scalable way. Read more in the introduction [blog post](https://www.openshift.com/blog/introducing-the-operator-framework?extIdCarryOver=true&sc_cid=701f2000001Css5AAC).
+
+## How does OLM enable cluster admins and developers?
+
+The OLM is enabled by default in OpenShift Container Platform 4.x, which aids cluster administrators in installing, upgrading, and granting access to Operators running on their cluster. The OpenShift Container Platform (OCP) web console provides management screens for cluster administrators to install Operators, as well as grant specific projects access to use the catalog of Operators available on the cluster.
+
+For developers, a self-service experience allows provisioning and configuring instances of databases, monitoring, and big data services without having to be subject matter experts, because the Operator has that knowledge baked into it.
+
+## Build with the Operator SDK
+
+[Operator SDK][osdk] is also a component of the [Operator Framework][operator-framework]. Writing an operator today can be difficult because of challenges such as using low-level APIs, writing boilerplate, and a lack of modularity which leads to duplication.
+The [Operator SDK][osdk] is a framework that uses the [controller-runtime][controller-runtime] library to make writing operators easier by providing:
+
+- High-level APIs and abstractions to write the operational logic more intuitively
+- Tools for scaffolding and code generation to bootstrap a new project fast
+- Extensions to cover common operator use cases
+
+Note that [Operator SDK][osdk] provides helpers and features to help you [integrate your project with OLM][olm-integration].
+
+## Package with the Operator Lifecycle Manager
```

Comment:
This should probably link to the Getting started page of this site. 
Packaging an operator for olm is quite an involved process that I'm not sure can be summarized in a paragraph. (this entire site is about packaging an operator for OLM)